Sources/SwiftParser/Types.swift

```diff
@@ -235,19 +235,50 @@ extension Parser {
   mutating func parseSimpleType(
     stopAtFirstPeriod: Bool = false
   ) -> RawTypeSyntax {
+    enum TypeBaseStart: TokenSpecSet {
```

Comment:
So much boilerplate :(

Reply:
Maybe a stupid question: is it possible to use one of the generated ones?

Reply:
I don’t think there’s any one that has these cases because we do walk into different parsing functions based on the token, similar to `ExpressionModifierKeyword`.